webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -7,11 +7,22 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
+      <base-button
+        v-if="showDeleteImageButton"
+        class="delete-image"
+        icon="close"
+        size="small"
+        circle
+        danger
+        filled
+        @click.prevent="deleteImage"
+      />
       <hc-teaser-image
         :contribution="contribution"
-        @addTeaserImage="addTeaserImage"
         :class="{ '--blur-image': form.blurImage }"
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-08T13:09:37Z" title="Wednesday, January 8th 2020, 2:09:37 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -198,6 +204,7 @@ export default {
       users: [],
       contentMin: 3,
       hashtags: [],
+      showDeleteButton: !!this.contribution,
```


The delete button should not be visible if there is no image. This happens if you edit a post which has no image:

<img width="1876" alt="Bildschirmfoto 2020-01-08 um 08 01 17" src="https://user-images.githubusercontent.com/25344101/71980448-53883880-3220-11ea-9c87-c414d53b1a8d.png">

And the delete button should not be visible if cropping is active:

<img width="1876" alt="Bildschirmfoto 2020-01-08 um 08 01 59" src="https://user-images.githubusercontent.com/25344101/71980595-b5e13900-3220-11ea-8179-2f1841acec9d.png">

And the delete button position should be the same as in the cropping, kind of.